src/fast_double_parser_c/fast_double_parser_c.h

```diff
@@ -28,32 +28,29 @@
  * limitations under the License.
  */
 
-#ifndef __FAST_DOUBLE_PARSER_H_C_H__
-#define __FAST_DOUBLE_PARSER_H_C_H__
+#pragma once
```

Comment:
@rafie i think its best not to change this code because its a "vendored" dependency and we want to keep it as close to original

Reply:
will revert.